client/app/components/OnboardingTopBar/OnboardingTopBar.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { PropTypes } from 'react';
+
+import css from './OnboardingTopBar.scss';
+
+class OnboardingTopBar extends React.Component {
+
+  t(translationKey) {
+    return this.props.translations[translationKey];
+  }
```

Comment:
If something like `utils.t(this.props.translations, translationKey)` is too much to type or if `f(translationKey) { return utils.t(this.props.translations, translationKey); }` still seems too much repetition, creating a [Higher Order Component](http://jamesknelson.com/structuring-react-applications-higher-order-components/) that "mixes in" the `this.t` function would be quite a clean solution to share this helper function.